### ISSUE TITLE
Remove title attribute on logo and site title + add aria-label attribute

### ISSUE THREAD

**Issue by ghost** [Nikeo/customizr#1719]
Question asked here : https://secure.helpscout.net/conversation/809522240/216714?folderId=607547

Why ? this doesn't add anything. We usually know that the site title or logo refers to home.
Websites usually don't display any title attribute on logo.

For accessibility, aria-label should be enough ? @eri-trabiccolo do you know anything about this ?

@eri-trabiccolo I assign this to you. Not urgent. When you'll have the time.

**Comment by eri-trabiccolo** [comment]
Yeah the title attribute is going to die. :D 
Looking at:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

Basically I think (for what I understand) the aria-label should not be added when displaying the site-title only, we should use the aria-labelledby (the link labelled by the site-title element) in that case.

**Comment by ghost** [comment]
OK thanks.
Let's go for those arias then.

**Comment by eri-trabiccolo** [comment]
So I'm going to use the aria-label for the link that wraps the logo, while no aria attributes are necessary for the site-title link, as the link text already says everything.
To be honest I'm not totally sure it is needed for the logo link, as the logo image has an alt...
